apps/backend/src/groups/groups.service.ts

```diff
@@ -25,6 +27,23 @@ export class GroupsService {
     return this.groupModel.count();
   }
 
+  async findByName(name: string): Promise<Group> {
+    return this.findOneBang({
+      where: {
+        name
+      }
+    });
+  }
+
+  async findOneBang(options: FindOptions | undefined): Promise<Group> {
```

Comment:
```suggestion
  async findOneBang(options: FindOptions?): Promise<Group> {
```

Reply:
This change appears to fail the build:

src/groups/groups.service.ts:31:56 - error TS2769: No overload matches this call.
  Overload 1 of 2, '(this: ModelStatic<Group>, options: NonNullFindOptions<any>): Promise<Group>', gave the following error.
    Argument of type 'FindOptions<any> | null' is not assignable to parameter of type 'NonNullFindOptions<any>'.
      Type 'null' is not assignable to type 'NonNullFindOptions<any>'.
  Overload 2 of 2, '(this: ModelStatic<Group>, options?: FindOptions<any> | undefined): Promise<Group | null>', gave the following error.
    Argument of type 'FindOptions<any> | null' is not assignable to parameter of type 'FindOptions<any> | undefined'.
      Type 'null' is not assignable to type 'FindOptions<any> | undefined'.

31     const group = await this.groupModel.findOne<Group>(options);

Reply:
That might be because I put the question mark in the wrong place haha.  Can you try with `findonebang(options?: findoptions)`